eureka-client-archaius2/src/main/java/com/netflix/appinfo/providers/Archaius2VipAddressResolver.java

```diff
@@ -0,0 +1,49 @@
+package com.netflix.appinfo.providers;
+
+import com.netflix.archaius.api.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Singleton
+public class Archaius2VipAddressResolver implements VipAddressResolver {
```

Comment:
Both archaius1 and archaius2 already support string interpolation.  This is code is completely unnecessary.